cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlMain.java

```diff
@@ -47,6 +48,14 @@ public static void main(String[] args) throws Exception {
         printErrorMessageAndDie();
       }
     }
+    // Future ssl bits
+    String protocol = "http";
```

Comment:
Nit: For clarity, would it be possible to move the hardcoded value to a variable? -- i.e. 
```
private static final String DEFAULT_PROTOCOL = "http";
...
String hostname = DEFAULT_PROTOCOL;
```

Reply:
I was following the approach taken for `port` variable. But, lemme fix all the relevant ones as well.